test/mux.sh

```diff
@@ -18,6 +18,12 @@ fi
 
 os::test::junit::declare_suite_start "test/mux"
 
+ARTIFACT_DIR=${ARTIFACT_DIR:-${TMPDIR:-/tmp}/origin-aggregated-logging}
+
+os::log::debug ARTIFACT_DIR is $ARTIFACT_DIR
+
+ops_cluster=${1:-"true"}
```

Comment:
The test should not rely on this. The test should do something like 
```
espod=$( get_es_pod es )
esopspod=$( get_es_pod es-ops )
if [ -n "$esopspod" ] ; then
  ops_cluster=true
else
  ops_cluster=false
  esopspod=$espod
fi
```

Reply:
Ok.  Being replaced.